Makefile

```diff
@@ -71,15 +71,15 @@ notes: check-bump
 	# Now generate the release notes to have them included in the release commit
 	towncrier build --yes --version $(UPCOMING_VERSION)
 	# Before we bump the version, make sure that the towncrier-generated docs will build
-	make build-docs
+	make check-docs
```

Comment:
It will be a little confusing to print out an empty preview of towncrier here, since all the newsfragments will be consumed by the build earlier.

Maybe this one should stay as `build-docs`, since it's too late to do any newsfragment validation anyway.